DigitalLearningSolutions.Web/Controllers/TrackingSystem/CourseSetup/AdminFieldsController.cs

```diff
@@ -298,6 +481,20 @@ private static bool TryGetAnswerIndexFromDeleteAction(string action, out int ind
             return int.TryParse(action.Remove(0, DeleteAction.Length), out index);
         }
 
+        private void SetViewModelCoursePromptNameOptions(AddAdminFieldViewModel model)
+        {
+            var coursePrompts = courseAdminFieldsService.GetCoursePromptsAlphabeticalList();
+            model.CoursePromptNameOptions =
+                SelectListHelper.MapOptionsToSelectListItems(coursePrompts, model.CustomPromptId);

```

Comment:
Just remembered why we didn't keep this on VM - it's because we don't want to save all these options on temp data (as we need to refetch them anyway). The options then were to:
- Have a different model for the data we store in TempData (I think we do this in registration).
- Not bother and use ViewBag

The first option is neater, but more dev, so I'll leave it up to you which method to use, but bare in mind that TempData is not meant for storing large quantities of data (we've had problems in the past for TempData exceeding the max and causing page to error)

Reply:
I'll just move it to the ViewBag as that's how registration prompts is done